cli/tools/jupyter/install.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::anyhow::bail;
+use deno_core::anyhow::Context;
+use deno_core::error::AnyError;
+use deno_core::serde_json;
+use deno_core::serde_json::json;
+use std::env::current_exe;
+use tempfile::TempDir;
+
+pub fn status() -> Result<(), AnyError> {
+  let output = std::process::Command::new("jupyter")
+    .args(["kernelspec", "list", "--json"])
+    .output()
+    .context("Failed to get list of installed kernelspecs")?;
+  let json_output: serde_json::Value =
+    serde_json::from_slice(&output.stdout)
+      .context("Failed to parse JSON from kernelspec list")?;
+
+  if let Some(specs) = json_output.get("kernelspecs") {
+    if let Some(specs_obj) = specs.as_object() {
+      if specs_obj.contains_key("deno") {
```

Comment:
Should we validate that this object matches what would be installed?

Reply:
I don't think it's needed, especially that it might change between jupyter versions.